templates/custom/header.tmpl

```diff
@@ -0,0 +1,14 @@
+	<script>

```

Comment:
we can not use custom

this has to go into https://github.com/go-gitea/gitea/blob/release/v1.13/templates/base/head.tmpl

Reply:
Custom is where Codeberg put it, makes sense that if it's built in to Gitea, that it's in base.

How do I give you permission to change these things? Allow you to push to my fork? Or do that myself? Or something else?